posthog/migrations/0404_remove_propertydefinition_property_type_is_valid_and_more.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 4.2.11 on 2024-04-21 21:11
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("posthog", "0403_plugin_has_private_access"),
+    ]
+
+    operations = [
+        migrations.RemoveConstraint(
```

Comment:
i think the drawback we have is even a canary deploy doesn't help us since it's the DB that we lock if something does go wrong

but I also don't think we have a good alternative here, this is what Django is going to do to make this change 🙈 

probably we need to merge this when the system isn't under load and with someone from infra on hand ready to help fix all the things if it does cause downtime.

Reply:
(and feel happily foolish if it goes in without incident :))